Zapper-fi/studio
# fix(studio): Mass revamp of KyberSwap Classic

## Description

- separate pools between KS (static fees) and DMM (dynamic fees)
- integrate farms v2 that use FairLaunch V2 model (multi-masterchef)

## Checklist


## How to test?

- pick the network of your choice in the list of available configs [from the KyberSwap repo](https://github.com/KyberNetwork/kyberswap-interface/tree/main/src/constants/networks)
- grab an address for FairLaunchV2 in the config, if it exists (example: `0xFFD22921947D75342BFE1f8efAcEE4B8B3b5183F` on Polygon)
- Use the appropriate network scanner to verify the pool length (it should be always 1) and pick the stakeToken address from `getPoolInfo(0)`
- This token is the classic farm LP token, so go grab a token holder randomly (e.g. `0xa2dfeb674d997b68ec5adb0a6fb9136bd45c2d2d`) and verify its balance using your local env URL: http://localhost:5001/apps/kyberswap-classic/balances?addresses[]=0xa2dfeb674d997b68ec5adb0a6fb9136bd45c2d2d&network=polygon
